experimental/dds/tree2/src/feature-libraries/editable-tree-2/README.md

```diff
@@ -0,0 +1,84 @@
+    This includes use-cases like passing the tree to generic javascript object processing libraries like `JSON.stringify`, nodes's `assert.deepEqual`, some structural clone implementations, [JMESPath](https://jmespath.org/) and many others.
+
+    Since this is an open ended set of preexisting third party libraries, compatibility is on a best effort basis.
+    However, a few things are taken into account:
+
+    -   The content visited by recursively walking over all the enumerable own properties should fully and unique describe the tree assuming the view schema is available (see "Javascript Object API: `enumerable` and `own` properties' below for details).
+        This requires some extra properties to prevent MapNodes and sequences from blocking the traversal, and provides guidance for which properties should be enumerable own properties.
+    -   Adapters can be provided to (lazily) view any subtree in a more JavaScript object like form, with different options to pick different trad-offs like boxed vs. unboxed and including getters (as non-enumerable properties) back to the editable view. (TODO: actually implement this):
+
+    Like with the generic tree readers above, this case intentionally does not provide editing.
+    Also like the generic tree readers case, it provides an easy way for code which does understand the schema to get back to the Schema Aware API for editing if needed.
+    (TODO: Add APIs for doing such casts safely.
+    Possible way to do instance of checks or symbols to recover which objects are nodes and fields)
+
+-   Subscribing to changes.
+    This includes cases like updating the user interface to reflect changes
+
+    This could be considered part of either reading API, but is being addressed separately here.
+    APIs for receiving these changes take the form of registering callbacks for events.
+    These events can be triggered by anything that impacts the content of the tree including local edits, remote edits, merges, rebasing local edits, transaction rollback, undo etc.
+    These can also be batching together, possibly eliminating intermediate states and reordering operations that don't impact each-other (like edits to different parts of the tree).
+    This means these event subscriptions should only be treated as changes to the content, and users of them should not attempt to ascribe meaning or providence to any particular event.
+
+    Some control over these events is available via branching which provides snapshot isolation (among other things):
+    this is done at a higher level than the tree API covered here but can be accessed via each tree entities context property. (TODO: provide this access).
+
+## Javascript Object API: `enumerable` and `own` properties
+
+See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Enumerability_and_ownership_of_properties for context.
+Note that `enumerable` has nothing to do with iterable: Iterable is not relevant to this section.
```

Comment:
Nit (since we put enumerable in backticks)
```suggestion
Note that `enumerable` has nothing to do with `iterable`: Iterable is not relevant to this section.
```

Reply:
this is using iterable as an English word, not strange JS standard jargon (like `enumerable`). We could refer to the typescript  `Iterable` or JS `Symbol.iterator` if we want to be more specific, and those could get back ticks.

Reply:
Oh, that's fair. Maybe adding an "(e.g. `Iterable` or `Symbol.iterator`) after the English "iterable" would help clarify a bit. But I don't think it's strictly needed. Either way.

Reply:
done.